skbio/io/tests/test_fasta.py

```diff
@@ -411,8 +411,10 @@ def test_fasta_to_generator_valid_files(self):
                 obs = list(_fasta_to_generator(fasta_fp, **kwargs))
                 self.assertEqual(len(obs), len(exp))
                 for o, e in zip(obs, exp):
-                    self.assertTrue(o.equals(e,
-                                             ignore=['positional_metadata']))
+                    # ignore positional metadata by creating a copy of the
+                    # expected sequence without positional metadata
+                    e = e._to(positional_metadata=None)
```

Comment:
Added a TODO to this occurrence. When we have `copy` we can search for all usages of `_to` and replace with `copy` and a setter.